TWLight/users/views.py

```diff
@@ -562,6 +562,12 @@ def delete(self, request, *args, **kwargs):
             user_authorization.date_expires = date.today() - timedelta(days=1)
             user_authorization.save()
 
+        # Delete bundle authorizations, if user is eligible for bundle
+        if user.editor.wp_bundle_authorized:
+            bundle_auths = user.editor.get_bundle_authorization
+            if bundle_auths is not None:
+                bundle_auths.delete()
+
```

Comment:
Agree with @Samwalton9 , something like this should do just fine.
```suggestion
        # Delete any bundle authorizations.
        bundle_auths = user.editor.get_bundle_authorization
            if bundle_auths:
                bundle_auths.delete()

```

Reply:
Done!